src/reaper_osara.cpp

```diff
@@ -3809,17 +3809,47 @@ void cmdDeleteTimeSig(Command* command) {
 		outputMessage(translate("time signature deleted"));
 }
 
-void cmdRemoveStretch(Command* command) {
-	MediaItem* item = GetSelectedMediaItem(0, 0);
-	if (!item)
-		return;
-	MediaItem_Take* take = GetActiveTake(item);
-	if (!take)
+void cmdhAddOrRemoveStretch(int command) {
+	const int itemCount = CountSelectedMediaItems(nullptr);
+	if (itemCount == 0) {
+		outputMessage(translate("no selected items"));
 		return;
-	int count = GetTakeNumStretchMarkers(take);
-	Main_OnCommand(41859, 0); // Item: remove stretch marker at current position
-	if (GetTakeNumStretchMarkers(take) != count)
-		outputMessage(translate("stretch marker deleted"));
+	}
+	int oldCount = 0;
+	for (int i = 0; i < itemCount; ++i) {
+		MediaItem* item = GetSelectedMediaItem(nullptr, i);
+		MediaItem_Take* take = GetActiveTake(item);
+		if (!take)
+			continue;
+		oldCount += GetTakeNumStretchMarkers(take);
+	}
+	Main_OnCommand(command, 0);
+	int newCount = 0;
+	for (int i = 0; i < itemCount; ++i) {
+		MediaItem* item = GetSelectedMediaItem(nullptr, i);
+		MediaItem_Take* take = GetActiveTake(item);
+		if (!take)
+			continue;
+		newCount += GetTakeNumStretchMarkers(take);
+	}
+	int difference = newCount - oldCount;
+	if (newCount >= oldCount) {
+		if (shouldReportTimeMovement()) {
+		// Translators: Reported when one or more stretch markers are added. {} will be replaced by the number of stretch markers, EG "2 stretch markers added".
+			outputMessage(format(
+				translate_plural("{} stretch marker added", "{} stretch markers added", difference),
+				difference));
+		}
+	} else {
+		// Translators: Reported when one or more stretch markers are removed. {} will be replaced by the number of stretch markers, EG "2 stretch markers removed".
+		outputMessage(format(
```

Comment:
Is it intentional that we respect shouldReportTimeMovement for addition, but not removal? I guess I kinda get this - you might well add them while playing, but you're less likely to remove them while playing, since you have to move to the marker first anyway - but I wanted to check because it does feel asymmetric.